ReflectometryServer/ChannelAccess/pv_manager.py

```diff
@@ -154,7 +154,7 @@ def _add_status_pvs(self):
         self._add_pv_with_fields(SERVER_MESSAGE, None, {'type': 'char', 'count': 400}, "Message about the beamline",
                                  PvSort.RBV, archive=True, interest="HIGH", on_init=True)
```

Comment:
This also looks to be a char waveform, does it need to be archived or is it stored elsewhere as well?

Reply:
This PV is mostly inteded to provide live feedback to the user so it does not need to be archived. I have changed it so it gets logged to the IOC log at least in case it could be useful for debugging after the fact.